python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -330,6 +337,14 @@ def predict(self, X, predict_model="GPU", algo='auto',
         delayed : bool (default = True)
             Whether to do a lazy prediction (and return Delayed objects) or an
             eagerly executed one.
+        broadcast_data : bool (default = False)
+            Usually the trees are merged in a single model for the workers
```

Comment:
Instead of "usually" can we say "If broadcast_data=False, the trees are merged ..."?
 (since whether it's usual or not depends on which flag you pass in)